### DISCUSSION TITLE
How to find icon codes

### DISCUSSION THREAD

**Discussion by rossmeyer1** [JanDeDobbeleer/oh-my-posh#2580] [Troubleshoot] [answered]
I don't know how to word this, but I'd like to find new icons and their corresponding codes.

E.g. \uf31b corresponds to the Ubuntu icon. But how would I search for new ones? I'd like to find a foot or trainer/sneaker icon, for example.

**Comment by JanDeDobbeleer** [comment]
You can make use of the Nerd Fonts [website](https://www.nerdfonts.com/cheat-sheet).

**Comment by rossmeyer1** [comment, marked as answer]
I've tried that, but it doesn't seem to show them in the /uf format.

Foot, for example is e241.

**Reply by JanDeDobbeleer** [reply]
@GHRoss correct, all you need to do is prefix with `\u`. So `e241` becomes `\ue241`.

**Comment by rossmeyer1** [comment]
Oh whoops, thank you!
